seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/Chrome.scala

```diff
@@ -1,23 +1,23 @@
 package org.scalajs.jsenv.selenium
 
-import org.openqa.selenium.chrome.ChromeDriverService
+import org.openqa.selenium.chrome.{ChromeDriverService, ChromeOptions}
 import org.openqa.selenium.remote._
 
-object Chrome extends SeleniumBrowser {
+class Chrome(chromeOptions: ChromeOptions = new ChromeOptions) extends SeleniumBrowser {
```

Comment:
This change is breaking binary compatibility and this is not following the same design as the rest of the classes.
I would suggest changing it to something like this:
```scala 
object Chrome {
  def apply(): Chrome = new Chrome(new ChromeOptions)
}

class Chrome private (chromeOptions: ChromeOptions) extends SeleniumBrowser {
  def name: String = "Chrome"
  def newDriver: BrowserDriver = new ChromeDriver
  def withChromeOptions(chromeOptions: ChromeOptions): Chrome = new Chrome(chromeOptions)
  private class ChromeDriver extends BrowserDriver {
    ...
  }
}
```

If this is changed you should also change a bit the `Firefox` one to have the same structure:
```scala
object Firefox {
  def apply(): Firefox = new Firefox
}

class Firefox private () extends SeleniumBrowser {
  def name: String = "Firefox"
  def newDriver: BrowserDriver = new FirefoxDriver
  private class FirefoxDriver extends BrowserDriver {
    protected def newDriver(): RemoteWebDriver =
      new org.openqa.selenium.firefox.FirefoxDriver()
  }
}
```